common/match/dissolve.ts

```diff
@@ -24,6 +25,14 @@ export async function dissolveMatch(match: Match, dissolveReason: number, dissol
             dissolveReason,
         },
     });
+    const matchLectures = await prisma.lecture.findMany({
+        where: {
+            matchId: match.id,
+        },
+    });
+    matchLectures.forEach(async (lecture) => {
+        await deleteZoomMeeting(lecture.zoomMeetingId);
```

Comment:
Shouldn't all lectures use the same meeting? If not, what happens if the same meeting is deleted twice?

Reply:
What happens if the zoomMeetingId is null?

Reply:
Not all lectures of a group or match use the same meeting due to possible conflicts with meetings occurring at the same time. Only weekly meetings that get created together receive the same meeting ID. 

If a meeting is deleted, it is gone. The response given from the zoom SDK would be something like "Meeting does not exist".

I changed the deleteZoomMeeting function, so it just returns when meetingId is null.